README.md

```diff
@@ -700,4 +708,4 @@ Please Tweet, star, and let any Elixir programmer know about
 [Elixir]: http://elixir-lang.org
 [Hex]: https://hex.pm/packages
 [license]: http://creativecommons.org/licenses/by/3.0/deed.en_US
-[Ruby community style guide]: https://github.com/bbatsov/ruby-style-guide
\ No newline at end of file
+[Ruby community style guide]: https://github.com/bbatsov/ruby-style-guide
```

Comment:
The new line at the end was removed. See: https://github.com/victorlcampos/elixir_style_guide/commit/387ee554a54f20e2c2aff3b276d7607fdcf594ec.patch

```diff
@@ -700,4 +708,4 @@ Please Tweet, star, and let any Elixir programmer know about
 [Elixir]: http://elixir-lang.org
 [Hex]: https://hex.pm/packages
 [license]: http://creativecommons.org/licenses/by/3.0/deed.en_US
-[Ruby community style guide]: https://github.com/bbatsov/ruby-style-guide
\ No newline at end of file
+[Ruby community style guide]: https://github.com/bbatsov/ruby-style-guide
```